client/gulpfile.js

```diff
@@ -67,6 +103,7 @@ gulp.task('watch', () => {
   })
   gulp.watch(sources.css.all, ['styles', 'stylelint'])
   gulp.watch(sources.scripts.all, ['scripts'])
+  gulp.watch(sources.images.svg.all, ['svgstore'])
```

Comment:
Get you. Perhaps a directory called 'icons' housing the svg files would be better?